Directly_import_images.md

```diff
@@ -0,0 +1,80 @@
+# In this tutorial we'll directly import images from publicly available online datasets into napari. 
```

Comment:
This heading reads like it's actually the first sentence of the tutorial? Can we make a shorter heading, then make this the first sentence?

In markdown, that would look like this:
```
# How to import images from online sources into napari

In this tutorial we'll directly import images from publicly available online datasets into napari. 
```